src/OSPSuite.UI/Views/Journal/JournalPageEditorView.cs

```diff
@@ -91,6 +91,9 @@ public void BindTo(JournalPageDTO journalPageDTO)
 
          _screenBinder.BindToSource(journalPageDTO);
 
+         //Ensure that the control is not marked as modified after binding. 
+         uxRichEditControl.Modified = false;
```

Comment:
This is the one variable that is sometimes true right after binding. This is used to check if something has changed before saving. I set it to false explicitly and all is well